lib/logstash/codecs/fluent.rb

```diff
@@ -22,15 +24,23 @@
 #
 # Notes:
 #
-# * the fluent uses a second-precision time for events, so you will never see
-#   subsecond precision on events processed by this codec.
+# * to handle EventTime msgpack extension, you must specify nanosecond_precision parameter as true.
 #
 class LogStash::Codecs::Fluent < LogStash::Codecs::Base
+  require "logstash/codecs/fluent/event_time"
```

Comment:
I had to change this to `require_relative "event_time"` to make it work on a new Logstash 6.2.2 setup otherwise you get `{:exception=>#<LoadError: no such file to load -- logstash/codecs/fluent/event_time>`

Reply:
Umm..., in my environemnt, `require_relative "fluent/event_time" and `require "logstash/codecs/fluent/event_time"` does not work. :'(

Reply:
My plugin setup procedure is:

```bash
$ gem build logstash-codec-fluent.gemspec
$ logstash-plugin install logstash-codec-fluent-*.gem
```

Reply:
Interesting. I'll do another test tomorrow and I'll let you know.